examples/nirfmxwlan/txp-basic.py

```diff
@@ -0,0 +1,222 @@
+                selector_string="",
+                iq_power_edge_source="0",
+                iq_power_edge_slope=nirfmxwlan_types.IQ_POWER_EDGE_TRIGGER_SLOPE_RISING_SLOPE,
+                iq_power_edge_level=-20.0,
+                trigger_delay=0.0,
+                trigger_min_quiet_time_mode=nirfmxwlan_types.TRIGGER_MINIMUM_QUIET_TIME_MODE_AUTO,
+                trigger_min_quiet_time_duration=5.0e-6,
+                iq_power_edge_level_type=nirfmxwlan_types.IQ_POWER_EDGE_TRIGGER_LEVEL_TYPE_RELATIVE,
+                enable_trigger=True,
+            )
+        )
+    )
+    raise_if_error(
+        client.CfgStandard(
+            nirfmxwlan_types.CfgStandardRequest(
+                instrument=instr, selector_string="", standard=nirfmxwlan_types.STANDARD_802_11_AG
+            )
+        )
+    )
+    raise_if_error(
+        client.CfgChannelBandwidth(
+            nirfmxwlan_types.CfgChannelBandwidthRequest(
+                instrument=instr, selector_string="", channel_bandwidth=20e6
+            )
+        )
+    )
+    if auto_level:
+        raise_if_error(
+            client.AutoLevel(
+                nirfmxwlan_types.AutoLevelRequest(
+                    instrument=instr, selector_string="", measurement_interval=10e-3
+                )
+            )
+        )
+    else:
+        raise_if_error(
+            client.CfgReferenceLevel(
+                nirfmxwlan_types.CfgReferenceLevelRequest(
+                    instrument=instr, selector_string="", reference_level=0.0
+                )
+            )
+        )
+    raise_if_error(
+        client.SelectMeasurements(
+            nirfmxwlan_types.SelectMeasurementsRequest(
+                instrument=instr,
+                selector_string="",
+                measurements=nirfmxwlan_types.MEASUREMENT_TYPES_TXP,
+                enable_all_traces=True,
+            )
+        )
+    )
+    raise_if_error(
+        client.TXPCfgMaximumMeasurementInterval(
+            nirfmxwlan_types.TXPCfgMaximumMeasurementIntervalRequest(
+                instrument=instr, selector_string="", maximum_measurement_interval=1e-3
+            )
+        )
+    )
+    raise_if_error(
+        client.TXPCfgAveraging(
+            nirfmxwlan_types.TXPCfgAveragingRequest(
+                instrument=instr,
+                selector_string="",
+                averaging_enabled=nirfmxwlan_types.TXP_AVERAGING_ENABLED_FALSE,
+                averaging_count=10,
+            )
+        )
+    )
+    raise_if_error(
+        client.Initiate(
+            nirfmxwlan_types.InitiateRequest(instrument=instr, selector_string="", result_name="")
+        )
+    )
+
+    ### Fetch results ###
+
+    txp_fetch_power_trace_response = raise_if_error(
+        client.TXPFetchPowerTrace(
+            nirfmxwlan_types.TXPFetchPowerTraceRequest(
+                instrument=instr, selector_string="", timeout=10.0
+            )
+        )
+    )
+    x0 = txp_fetch_power_trace_response.x0
```

Comment:
These are never used

Reply:
Agreed that they aren't used. But as an example script, I think it's useful to leave them in, so users can see that data like this is available.